docs/writing-rules/generic-pattern-matching.md

```diff
@@ -11,9 +11,9 @@ description: "Semgrep can match generic patterns in languages that it doesn’t
 
 ## Introduction
 
-Semgrep can match generic patterns in languages that it does **not** yet support. Use generic pattern matching for languages that do not have a parser, configuration files, or other structured data such as XML.
+Semgrep can match generic patterns in languages that it does **not** yet support. Use generic pattern matching for languages that do not have a parser, configuration files, or other structured data such as XML. Generic pattern matching can also be useful in files with mixed content even if the languages are supported, such as HTML with embedded JavaScript or PHP code. In those cases you can also consider [Extract mode (experimental)](/docs/writing-rules/experiments/extract-mode/), but generic patterns may be simpler and still effective.
```

Comment:
"Mixed content" means a file with multiple languages used in the file, right? (Perhaps it would be helpful to add a brief definition here)

Reply:
Yeah, good point - I'm not really trying to use or define a term, so I was hoping the examples would work. Maybe I should use something like "in files containing code in more than one supported language".

Reply:
Yeah, I like that!

Reply:
Perfect, I updated. Appreciate the review! I'm interested in having someone from PA or SR review as well, so will wait a bit on merge.